src/main/java/uk/gov/justice/digital/delius/data/api/CustodialStatus.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+@Getter
+@Builder
+public class CustodialStatus {
```

Comment:
I know it's really hit and miss in the codebase, but do we need any @ApiModelProperty documentation on these?

Reply:
Will review and add as appropriate, don't want to go crazy if names are self explanatory but there may well be non-obvious things that are worth documenting so good call.